File: python_files/ex10.py
class Calculadora:
    def somar_dois(self, a, b):
        return a+b
    
    def somar_tres(self, a, b, c):
        return a+b+c
    
calc = Calculadora()

# Usando o método para somar dois números
soma_dois = calc.somar_dois(5, 10)
print(f"Soma de dois números: {soma_dois}")

# Usando o método para somar três números
soma_tres = calc.somar_tres(5, 10, 20)
print(f"Soma de três números: {soma_tres}")

File: go_files/ex10.go
package main

import "fmt"

func somarDois(a int, b int) int {
	return a + b
}

func somarTres(a int, b int, c int) int {
	return a + b + c
}

func main() {
	// Usando a função para somar dois números
	somaDois := somarDois(5, 10)
	fmt.Printf("Soma de dois números: %d\n", somaDois)

	// Usando a função para somar três números
	somaTres := somarTres(5, 10, 20)
	fmt.Printf("Soma de três números: %d\n", somaTres)
}